README.md

```diff
@@ -48,7 +48,7 @@ The major components of the Boson Project are:
 
 If you don't already have an OpenShift cluster with Serverless installed, or a
 Kubernetes cluster with Knative Serving and Eventing, you can follow our
-[Getting Started with Kubernetes Guide](https://github.com/boson-project/faas/blob/develop/docs/getting_started_kubernetes.md)
+[Getting Started with Kubernetes Guide](https://github.com/boson-project/func/blob/develop/docs/getting_started_kubernetes.md)
 to provision a cluster for function deployment. Or for a quick and easy set up,
 follow our instructions for setting up a [kind cluster]('kind-setup.md') on your
```

Comment:
The apostrophes shouldn't be here.

Reply:
The link works, but i'll change it

Reply:
Interesting at least richdiff leads to https://github.com/boson-project/functions/blob/master/'kind-setup.md' which is not working for me, but it might be a bug of richdiff.